lib/services/amazonka-route53/src/Amazonka/Route53/Internal.hs

```diff
@@ -64,8 +64,8 @@ instance FromXML ResourceId where
 getHostedZoneId :: Region -> Maybe ResourceId
 getHostedZoneId = \case
     Ohio            -> Just "Z2O1EMRO9K5GLX"
-    NorthVirginia   -> Just "Z3AQBSTGFYJSTF "
-    NorthCalifornia -> Just "Z2F56UZL2M1ACD "
```

Comment:
Is this change related to the above change?

Reply:
No:

> Branch is currently based off #723 so I can write a decent changelog entry. Merge that first and then this will look sensible.